src/ImageRunModal.jsx

```diff
@@ -119,18 +120,37 @@ const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount }) =>
         </Grid>
     );
 
+const handleEnvValue = (key, value, idx, onChange, additem, itemCount) => {
+    // Allow the input of KEY=VALUE seperated value pairs for bulk import
+    if (value.includes('=')) {
+        const parts = value.trim().split(" ");
```

Comment:
Why splitting on space? Shouldn;t we split on new line? (although it works)

Reply:
It seems that the browser for a single line input converts newlines to spaces.